tests/resources/VCH-Util.robot

```diff
@@ -303,3 +303,18 @@ Cleanup Dangling Containers On Test Server
     \   ${name}=  Fetch From Right  ${vm}  /
     \   ${out}=  Run  govc datastore.rm ${name}
     \   Wait Until Keyword Succeeds  6x  5s  Check Delete Success  ${name}
+
+
+Get VCH IP
```

Comment:
We have (or had) an existing drone library call for this: `Get Docker Params`
It's specifically for parsing the install output, but it's the same exercise that we're doing here.